xmtp/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 pub mod account;
 pub mod builder;
 pub mod client;
+pub mod in_memory_persistence;
```

Comment:
may want to add this file to the persistence module rather than creating a new one.